code/modules/mob/mob_grab.dm

```diff
@@ -433,9 +435,10 @@
 
 /obj/item/grab/Destroy()
 	if(affecting)
-		affecting.pixel_x = 0
-		affecting.pixel_y = 0 //used to be an animate, not quick enough for del'ing
-		affecting.layer = initial(affecting.layer)
+		if (!affecting.buckled)
```

Comment:
Can you check what happens if you grab somebody unbuckled. Then buckle them and release the grab?

Reply:
![image](https://user-images.githubusercontent.com/1496804/84190557-6cc73680-aa97-11ea-9571-bc86d00c32f0.png)

Oof. The most obvious fix would be to reset the pixel offsets on buckling while there's a grab, but I opted to remove the grab when buckling someone, like grabbing a mob you're pulling does. RP-speaking, it doesn't make sense to be able to buckle someone while you're strangling them.